### ISSUE TITLE
[Suggestion] order tabs by last opened first

### ISSUE THREAD

**Issue by neeraj-htp** [alyssaxuu/omni#15]
Can we have tabs' ordering by last open at the top, same as chrome tabs search do? It will be a huge boost in productivity. While working people usually keep switching between 3, 4 tabs, putting them on top of the list makes it much much easier to navigate.

**Comment by alyssaxuu** [comment]
Hmm I'm not sure if it might be confusing though, right now it's ordered the opposite way so it follows how the tabs are displayed in Chrome, so you might have an idea of where they are based on the favicon. Whereas your way you'd have to remember how recently you opened the tab. I don't know, maybe I'm just a much more visual person xP

Open to changing it if people prefer it this way though!

**Comment by hellokwad** [comment]
I agree with @neeraj-htp, we don't need to really remember how recently we've opened the tab because it's an order based on activity, so the most relevant ones will be on top and that will help switch faster. It would also help for switching back and forth between 2 tabs. Something we often have to do while working. :)
Having tabs ordered like in Chrome isn't an added value, we can just see our tabs and switch to it in Chrome, IMO.
